php-nginx/entrypoint.sh

```diff
@@ -138,4 +134,7 @@ chmod -R 550 ${DOCUMENT_ROOT}
 # Change the www-data's shell back to /usr/sbin/nologin
 chsh -s /usr/sbin/nologin www-data
 
+# Remove the loose php-cli.ini
+rm ${PHP_DIR}/lib/php-cli.ini
```

Comment:
@bshaffer 

Basically, adding `display_errors=STDERR` doesn't improve debugging experience with suhosin enabled.

I would suggest that we rollback the last commit and provide a script for better debugging in another PR. WDYT?